### PR TITLE
Solved final logic of clicked liar button and winning game logic

### PR DESCRIPTION
**Pull request by hhshan21**
Completed liar button function and winning game logic! Heavy PR but was the last function to settle before game was completed.

### CODE REVIEW

File: Main/game/game.js [hunk 1]
```diff
@@ -146,24 +136,22 @@ export default class Game {
...
-                </div> 
-            `;
-            computerBid.innerHTML = `
-                <button id="liar">Liar</button>
-            `;
-            showBidOptions.style.display = 'flex';
-        }, 3000)
-
-        this.showPlayerBidChoices();
+        const dJ = this.#players[1].name;
```
Comment (tlkiong) [inline Main/game/game.js]:
```suggestion
        const { name } = this.#players[1].name;
```
why not just like this instead?

File: Main/game/game.js [hunk 2]
```diff
@@ -191,17 +179,25 @@ export default class Game {
     }
 
     clickLiar() {
+        const clickLiarBtn = document.querySelector('#liar');
+
+        let dJCont = document.getElementById('players-not-me').getElementsByClassName('dice-container')[0]
```
Comment (tlkiong) [inline Main/game/game.js:184]: Why `let`?

File: Main/game/game.js [hunk 3]
```diff
@@ -191,17 +179,25 @@ export default class Game {
...
+        
+        console.log('this.#players[1].dice[4].face: ', this.#players[1].dice[4].face)
```
Comment (tlkiong) [inline Main/game/game.js:186]: Why have console.log here?

**Review by tlkiong — APPROVED** [review]
Aside from some small mini comments, LGTM 🚀

File: Main/game/game.js (hunk 2 above)
Comment (hhshan21) [reply Main/game/game.js:184]: Will amend to const

File: Main/game/game.js (hunk 3 above)
Comment (hhshan21) [reply Main/game/game.js:186]: was a work check. Will remove.

File: Main/game/game.js (hunk 1 above)
Comment (hhshan21) [reply Main/game/game.js]: ok will amend!

**Pull request merged**